datalab/mlalpha/_cloud_predictor.py

```diff
@@ -32,29 +32,22 @@
 class CloudPredictor(object):
   """Preforms cloud predictions on given data."""
 
-  def __init__(self, model_name, version_name, metadata_path=None, label_output=None,
+  def __init__(self, model_name, version_name, label_output=None, 
```

Comment:
The 'label_output' keyword parameter is now unused.

I think that either the 'self._label_output = label_output' line was removed on accident, or this should be removed from the constructor (possibly replaced with an '**unused_kwargs' parameter?).